ldk/javascript/example/self-test-loop/generate-banner.js

```diff
@@ -0,0 +1,18 @@
+function generateMetadata() {
```

Comment:
I thought, this banner should be in the javascript ldk and not in the loop.

Reply:
That's been removed at this point